docs/about-ensembl/annotation-and-prediction/chromosomes-scaffolds-contigs.md

```diff
@@ -0,0 +1,20 @@
+---
+slug: chromosomes-scaffolds-contigs
+title: Chromosomes scaffolds and contigs
+description: The relationship between contigs, scaffolds and chromosomes in Ensembl 
+---
+
+Genome assemblies are hierarchical. The shortest assembly components are contigs, which are sequences taken from individuals. Contigs are assembled into longer scaffolds, and scaffolds are assembled into chromosomes if there is sufficient mapping information. Many genome assemblies have only been assembled to the scaffold level.
+
+Scaffolds are classified in three ways:
+
+# Placed scaffolds: 
```

Comment:
A `#` in markdown will create an `h1` heading in html, which will be the `h1` heading in html. You probably want a second-level heading here, which is represented in markdown as `##`

![image](https://user-images.githubusercontent.com/6834224/188880939-07c1904b-57d7-49a7-ae30-4125b8ba200b.png)

Also, why does this heading end in a colon while others don't?